### DISCUSSION TITLE
Notally 6.0-RC1

### DISCUSSION THREAD

**Discussion by Crustack** [Crustack/Notally#27] [Announcements]
### Subtasks
* In Lists you can now swipe tasks to the right to group them (the delete button is moved to the right)
* New preference for setting auto-sort for Tasks, automatically moves checked items to the bottom
* New Toolbar entries to un-/check all items and to remove all checked Tasks
* The date format setting is used for the notes preview as well as notes edit view and the widgets

https://github.com/user-attachments/assets/2402e17b-2e50-4ec5-89e9-198e95f41c60

### Attach files to note
Besides images you can now attach any file to a note (will also be copied into backups)

[notally_file_attachments.webm](https://github.com/user-attachments/assets/f29d4f29-55fd-4061-a27f-4bdd9becadec)

### Change History
* Undo/Redo buttons in the toolbar to undo/redo changes to the list, such as add/delete tasks, text changes, etc.
* ATM most changes work reliably, some are still buggy, working on it

<hr /><em>This discussion was created from the release <a href='https://github.com/PhilKes/Notally/releases/tag/v6.0-RC1'>Notally 6.0-RC1</a>.</em>

**Comment by ippocratis** [comment]
The uploaded apk is likely not signed.

If rooted someone could manually sign it

https://github.com/PhilKes/Notally/pull/24#issuecomment-2397577509

Other than that I love the file attachment feature .

Thank you so mutch for this.

I checked the export/import feature and the attachments are preserved fine , deleted the original files from system and notallyx still restored the attachments fine.
